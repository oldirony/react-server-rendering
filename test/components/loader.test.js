import React from 'react';
import Loader from '../../src/components/loader/loader';

test('It renders correcly', ()=>{
	let loader = shallow(<Loader isVisible={false}/>);

	expect(loader).toMatchSnapshot();

	loader = shallow(<Loader isVisible={true} message={'Doing things'}/>);

	expect(loader).toMatchSnapshot();
})