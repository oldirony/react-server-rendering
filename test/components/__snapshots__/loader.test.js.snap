// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders correcly 1`] = `
<div
  className="loader"
>
  <div
    className="text"
  />
</div>
`;

exports[`It renders correcly 2`] = `
<div
  className="loader isVisible"
>
  <div
    className="text"
  >
    Doing things
  </div>
</div>
`;
